client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,1316 @@
+					},
+				},
+			},
+			{
+				featureName: 'Custom domain name',
+				tooltip:
+					'Remove “.wordpress.com” from your site address with a custom domain of your choosing. Free for one year (will renew at its regular price).',
+				planOne: {
+					annually: {
+						included: true,
+						copy: 'Free for the first year',
+						mobileCopy: 'Free domain for the first year.',
+					},
+					monthly: {
+						included: false,
+						copy: 'Included with annual plans',
+						mobileCopy: null,
+					},
+				},
+				planTwo: {
+					annually: {
+						included: true,
+						copy: 'Free for the first year',
+						mobileCopy: 'Free domain for the first year',
+					},
+					monthly: {
+						included: false,
+						copy: 'Included with annual plans',
+						mobileCopy: null,
+					},
+				},
+				planThree: {
+					annually: {
+						included: true,
+						copy: 'Free for the first year',
+						mobileCopy: 'Free domain for the first year',
+					},
+					monthly: {
+						included: false,
+						copy: 'Included with annual plans',
+						mobileCopy: null,
+					},
+				},
+			},
+			{
+				featureName: 'SEO',
+				tooltip:
+					'SEO (Search Engine Optimization) helps your site rank higher, and more accurately in search engines, so more people can find your site.',
+				planOne: {
+					annually: {
+						included: true,
+						copy: 'Simple',
+						mobileCopy: 'Simple SEO',
+					},
+					monthly: {
+						included: true,
+						copy: 'Simple',
+						mobileCopy: 'Simple SEO',
+					},
+				},
+				planTwo: {
+					annually: {
+						included: true,
+						copy: 'Professional',
+						mobileCopy: 'Professional SEO',
+					},
+					monthly: {
+						included: true,
+						copy: 'Professional',
+						mobileCopy: 'Professional SEO',
+					},
+				},
+				planThree: {
+					annually: {
+						included: true,
+						copy: 'Professional',
+						mobileCopy: 'Professional SEO',
+					},
+					monthly: {
+						included: true,
+						copy: 'Professional',
+						mobileCopy: 'Professional SEO',
+					},
+				},
+			},
+			{
+				featureName: 'Customer support',
+				tooltip:
+					'Keep moving forward with your site at any time. Our Happiness Engineers will help you via email, live chat, or 24/7 priority live chat, depending on the plan you choose.',
+				planOne: {
+					annually: {
+						included: true,
+						copy: 'Live chat and email support',
+						mobileCopy: 'Live chat and email support',
+					},
+					monthly: {
+						included: true,
+						copy: 'Email support',
+						mobileCopy: 'Email support',
+					},
+				},
+				planTwo: {
+					annually: {
+						included: true,
+						copy: '24/7 priority live chat',
+						mobileCopy: '24/7 Priority live chat',
+					},
+					monthly: {
+						included: true,
+						copy: 'Email support',
+						mobileCopy: 'Email support',
+					},
+				},
+				planThree: {
+					annually: {
+						included: true,
+						copy: '24/7 priority live chat',
+						mobileCopy: '24/7 Priority live chat',
```

Comment:
Is the `P` here a capitalization error or intentional? I'm thinking it should be just `priority` like above, but am not sure :)